resources/user-agents/browsers/puffin/puffin-2-0.json

```diff
@@ -28,45 +28,81 @@
       },
       "children": [
         {
-          "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) *Chrome/* Safari/* Puffin/#MAJORVER#.*AP Mobile",
+          "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) *Chrome/* Safari/* Puffin/#MAJORVER#.*",
```

Comment:
This is the same as line 50 now. Please leave the `AP` token or remove one of these two lines